src/base/http/types.h

```diff
@@ -70,6 +70,8 @@ namespace Http
     const char CONTENT_TYPE_FORM_ENCODED[] = "application/x-www-form-urlencoded";
     const char CONTENT_TYPE_FORM_DATA[] = "multipart/form-data";
```

Comment:
Offtopic:
We probably want to declare all such "public" constants as `inline` to avoid data duplication.

Reply:
Added another commit for it. It covers most portion of webAPI usage.

Reply:
>Added another commit for it. It covers most portion of webAPI usage.

👍